proto/rpc/v1/ledger_objects.proto

```diff
@@ -0,0 +1,141 @@
+syntax = "proto3";
+
+package rpc.v1;
+
+import "rpc/v1/amount.proto";
+
+message LedgerObject {
```

Comment:
Can you explain what you mean by the actual size limits of the fields?

Reply:
@MarkusTeufelberger Can you explain what you mean here?

Reply:
Often fields are just "bytes" (e.g. `bytes root_index = 2;`) - it might be useful to have a comment of `// 32 bytes fixed length` there, or even typing it a bit stronger by having a "Hash256" type instead (not always possible). Some other fields like Memos have a certain range of valid sizes - also there it might be useful to have this spelled out in the proto file that one needs to use in a client implementation anyways rather than just in the documentation all over xrpl.org.